src/showcase/showcase-changes.less

```diff
@@ -179,11 +179,15 @@
     }
 }
 
+.multi.rsi.indicator {
+    stroke: @tertiary-purple;
+}
```

Comment:
I couldn't find any references to `@tertiary-purple`, and for some reason, I don't think Less is overriding it (or the selector's changed in Bitflux). I'll have a look